test/Sia.Gateway.Tests/Requests/Events/CreatLinksWorksCorrectly_WhenCalledByEventsMethods.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Sia.Gateway.Controllers;
+using Sia.Shared.Protocol;
+using System.Collections.Generic;
+
+namespace Sia.Gateway.Tests.Requests.Events
+{
+    [TestClass]
+    public class GenerateLinksHeaderTestsForEvents
+    {
+        private List<string> methods;
+        private List<object> ids;
+        private Mock<IUrlHelper> urlHelperMock;
+        public GenerateLinksHeaderTestsForEvents()
+        {
+            // Arrange
+            methods = new List<string>();
+            ids = new List<object>();
+            urlHelperMock = new Mock<IUrlHelper>();
+            urlHelperMock.Setup(link => link.Link(It.IsAny<string>(), It.IsAny<object>()))
+                .Callback<string, object>(
+                    (s, o) =>
+                    {
+                        methods.Add(s);
+                        ids.Add(o);
+                    }
+                );
+
+        }
+
+        [TestMethod]
+        public void CreateLinksWorksCorrectly_WhenCalledInEventsMethods()
+        {
+            //Arrange
+            methods.Clear();
+            ids.Clear();
+            var eventsController = new EventsController(null, null, null, urlHelperMock.Object);
+            // Act
+            eventsController.CreateLinks("1", "2", null,null,"");
```

Comment:
Use named arguments in method invocation, as the order of arguments doesn't seem to indicate anything.